src/main/java/org/sagebionetworks/web/client/SynapseClient.java

```diff
@@ -372,7 +375,9 @@ EntityBundlePlus getEntityBundlePlusForVersion(String entityId, Long versionNumb
 
 	Entity moveEntity(String entityId, String newParentEntityId) throws RestServiceException;
 
-	String getUserIdFromUsername(String username) throws RestServiceException;
+    Team getTeam(String teamId) throws RestServiceException;
```

Comment:
Whitespace is off for these (if these are being cleaned up, then this should also be right).

Reply:
I fixed all my files to be indented with tabs. (Sometimes using spaces for alignment). I wish there was a way to show github diffs with tabsize = 4 instead of 8, because that's what I use in my IDE (Intellij IDEA).